### PR TITLE
Ingest Tweets and, process these and produce message in database

### PR DESCRIPTION
**Pull request by jecos**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -x
+set -e 
+cd harmonize \
+    && sbt assembly \
```
Comment (latch2112) [inline build.sh:5]: image

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    && cd ..
+
+docker-compose build
```
Comment (latch2112) [inline build.sh:8]: image

File: README.md [hunk 3]
```diff
@@ -1,21 +1,27 @@
...
-* Java or Scala using Apache Spark or Kafka Streams
-* Command-line user interface
+### HOW TO
+1. Import the Postman collection, and define these variables in an environment or in global. More information [on Postman website](https://learning.getpostman.com/docs/postman/environments_and_globals/intro_to_environments_and_globals/)  :
+* twitter_access_token
+* twitter_access_token_secret
+* twitter_consumer_key
+* *twitter_consumer_secret
+2. Run build.sh script
+3. Run docker-compose up
```
Comment (latch2112) [inline README.md:16]: image

File: build.sh (hunk 1 above)
Comment (jecos) [reply build.sh:5]: Missing Confluent resolver in build.sbt. Should be fine now.

**Comment by latch2112** [comment]
👍

**Pull request closed without merging**